project/src/main/world/environment/poki/poki-crowd.gd

```diff
@@ -0,0 +1,82 @@
+tool
+extends OverworldObstacle
+## Poki desert crowd member which appears in the overworld.
+##
+## This script randomizes the crowd member's color, appearance and direction.
+
+const CROWD_COLORS := [
+	Color("3d291f"),
+	Color("4c3125"),
+	Color("5d3b2b"),
+	Color("6c4331"),
+]
+
+## Current frame to display from the sprite sheet.
+export (int) var frame: int setget set_frame
+
+## If true, the sprite's texture is flipped horizontally.
+export (bool) var flip_h: bool setget set_flip_h
+
+## An editor toggle which randomizes the obstacle's appearance
+export (bool) var shuffle: bool setget set_shuffle
+
+export (int, 0, 3) var crowd_color_index: int setget set_crowd_color_index
+
+onready var _sprite := $Sprite
+onready var _wiggle_timer := $WiggleTimer
```

Comment:
Document this 'wiggle timer' field

Reply:
Added comment, `A timer which makes the crowd member animate slightly, alternating between two frames`